### PR TITLE
Github Pages Installer

### PR DESCRIPTION
**Pull request by jordan-rash**
add installer to github pages for easy button

### CODE REVIEW

**Comment by jordan-rash** [comment]
Confirmed this works as expected on `jordan-rash/nex`

File: README.md [hunk 1]
```diff
@@ -14,7 +16,7 @@ The easiest way to get started with Nex is to check out our [Using Nex](https://
 If you're already familiar with Nex and how it works, then you can get going quickly by installing the Nex CLI:
 
 ```
-go install github.com/synadia-io/nex/nex@latest
+curl -sSf https://nex.synadia.com/install.sh | sh 
 ```
 
 _**Note** that an installation method that doesn't require Go is coming soon._
```
Comment (autodidaddict) [inline README.md]: we should probably remove this note since we're adding the non-Go installation method :)

**Review by autodidaddict — CHANGES_REQUESTED** [review]
one tiny nit

**Review by autodidaddict — APPROVED** [review]
We'll want to make sure we update docs.nats.io 's getting started guide, too

**Pull request merged**